account_manager/src/common.rs

```diff
@@ -37,3 +39,61 @@ pub fn base_wallet_dir(matches: &ArgMatches, arg: &'static str) -> Result<PathBu
         PathBuf::new().join(".lighthouse").join("wallets"),
     )
 }
+
+pub fn strip_off_newline_codes(bytes: &mut Vec<u8>) -> Vec<u8> {
+    let mut strip_off = 0;
+    for (i, byte) in bytes.iter().rev().enumerate() {
+        if *byte == NEWLINE_SLASH_N || *byte == NEWLINE_SLASH_R {
+            strip_off = i + 1;
+        } else {
+            break;
+        }
+    }
+    bytes.resize(bytes.len() - strip_off, 0);
```

Comment:
Using [`Vec::truncate`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.truncate) would be slightly cleaner here.